CONTRIBUTING.md

```diff
@@ -1,15 +1,13 @@
 # Contributing
 
-You are thinking about contributing to Testcontainers? Awesome, it’s absolutely appreciated. When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change. Or pick an already existing issue.
+Please see the [main contributing guidelines](./docs/contributing.md).
 
-## Guidelines
+There are additional docs describing [contributing documentation changes](./docs/contributing_docs.md).
```

Comment:
I think this contributing docs is missing

Reply:
Thanks. I will add it. I created the PR by mistake aynway 😬